IBX-11536: MCP Servers (#3106)
ezsystems/developer-documentation: docs/ai/ai.md

---
description: AI interactions with [[= product_name =]]
page_type: landing_page
month_change: true
---

# Artificial Intelligence (AI)

> [[= product_name =]] embed AI capabilities, for example,

Reviewer: ```suggestion
[[= product_name =]] embeds AI capabilities, for example,
```

> to make recommendations to product customers and content readers with [Raptor connector][[#(raptor_connector_guide.md)#]],

Reviewer: Incorrect link

Author: Nice! I totally forgot I commented this one to avoid crash when it wasn't merged yet.
```suggestion
to make recommendations to product customers and content readers with [Raptor connector](raptor_connector_guide.md),
```

or to help editors in the back office with [AI Actions](ai_actions_guide.md).

> It's also opened to external AI capabilities with the exposition of [Model Context Protocol (MCP) servers](mcp_guide.md) to allow agents to interact with the system.

Reviewer: ```suggestion
It's also open to external AI capabilities through [Model Context Protocol (MCP) servers](mcp_guide.md), allowing agents to interact with the system.
```

---
description: AI interactions with [[= product_name =]]
page_type: landing_page
month_change: true
---

> # Artificial Intelligence (AI)

Reviewer 2: ```suggestion
# Artificial Intelligence
```

[[= product_name =]] embeds AI capabilities, for example,
to make recommendations to product customers and content readers with [Raptor connector](raptor_connector_guide.md),

> or to help editors in the back office with [AI Actions](ai_actions_guide.md).

Reviewer 2: ```suggestion
[[= product_name =]] includes built-in AI capabilities.
For example, it can provide recommendations to product customers and content readers with the [Raptor connector](raptor_connector_guide.md), and assist editors in the back office with [AI Actions](ai_actions_guide.md).
```

> It's also open to external AI capabilities through [Model Context Protocol (MCP) servers](mcp_guide.md), allowing agents to interact with the system.

Reviewer 2: ```suggestion
The platform is also open to external AI integrations through [MCP (Model Context Protocol) servers](mcp_guide.md), which allow AI agents to interact with the system in a standardized way.
```

> It's extensible. For example, you can implement [new AI actions](extend_ai_actions.md) or [new MCP server capabilities](mcp_usage.md).

Reviewer 2: ```suggestion
AI solutions are extensible. You can create [custom AI actions](extend_ai_actions.md) or expose [new MCP server capabilities](mcp_usage.md).
```

> To go further, an AI agent can learn how to use the [REST API](rest_api_usage.md) or the [GraphQL API](graphql.md).

Reviewer 2: ```suggestion

AI integration goes even further:

- Some AI agents can learn how to use the [REST](rest_api_usage.md) or [GraphQL](graphql.md) APIs.
```

> Agents (like the ones integrated in IDEs) can even learn the [PHP API](php_api.md) and help you in your development.

Reviewer 2: ```suggestion
- Other, like those integrated into IDEs, can even learn how to use the [PHP API](php_api.md) and assist you in code development.
```

---
description: AI interactions with [[= product_name =]]
page_type: landing_page
month_change: true
---

# Artificial Intelligence

[[= product_name =]] includes built-in AI capabilities.
For example, it can provide recommendations to product customers and content readers with the [Raptor connector](raptor_connector_guide.md), and assist editors in the back office with [AI Actions](ai_actions_guide.md).
The platform is also open to external AI integrations through [MCP (Model Context Protocol) servers](mcp_guide.md), which allow AI agents to interact with the system in a standardized way.
AI solutions are extensible. You can create [custom AI actions](extend_ai_actions.md) or expose [new MCP server capabilities](mcp_usage.md).

AI integration goes even further:

- Some AI agents can learn how to use the [REST](rest_api_usage.md) or [GraphQL](graphql.md) APIs.

> - Other, like those integrated into IDEs, can even learn how to use the [PHP API](php_api.md) and assist you in code development.

Reviewer 2: ```suggestion
- Other, like those integrated into IDEs, can learn how to use the [PHP API](php_api.md) and assist you in code development.
```
